submissions/ArthurGorbenko/html.css.practice/index.html

```diff
@@ -0,0 +1,178 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+<meta charset="utf-8">
+
+<title>Html/css practice</title>    
```

Comment:
Add meta tag for mobile viewport and Edge support.

Reply:
Yep,thanks.

Reply:
Yes)